### ISSUE TITLE
init_printing should change latex color if qtconsole background is black

### ISSUE THREAD

**Issue by asmeurer** [sympy/sympy#10655]
See for instance https://stackoverflow.com/questions/35539492/change-the-color-of-latex-output-in-ipython-jupyter.

Probably easy to fix.

**Comment by siddharthist** [comment]
Also see your answer at http://stackoverflow.com/questions/18257270/control-latex-expression-color-in-ipython-qtconsole :smiley:

**Comment by SalilVishnuKapur** [comment]
Is this where i need to look for fixing this sympy/sympy/printing/printer.py ? If not then kindly guide as I want to explore this.

**Comment by asmeurer** [comment]
`init_printing` is in sympy/interactive/printing.py.

**Comment by lapa19** [comment]
We can change back-color parameter to 'White' .. It will be readable irrespective of the background color, right ?

**Comment by asmeurer** [comment]
Yes, White would be the best color I'd say.

**Comment by sampadsaha5** [comment]
@asmeurer I have tried a fix, but  I had a doubt . 
If we have to access the background color variable of qtconsole do we need to import the jupyter library?

**Comment by sampadsaha5** [comment]
@lapa19 But there will be a problem if the background itself is light color.

**Comment by asmeurer** [comment]
Yes, you need to get the color information from Jupyter somehow (I don't know how to access it, but I'm sure it's doable).

**Comment by sampadsaha5** [comment]
Yah , I have found a fix for that but I was confused . I am creating a PR then we can discuss.

**Comment by sampadsaha5** [comment]
@asmeurer [Here](https://github.com/jupyter/qtconsole/blob/master/qtconsole/styles.py#L32-L43) The whole entries are kept into a string. So how can we extract the value of `background-color`. Do we have to use regular expressions or we can do it in other way also?

**Comment by asmeurer** [comment]
There's probably a better way to detect it.

**Comment by lapa19** [comment]
I tried changing the back-color parameter to White . Here are the results for light and dark backgrounds:

Dark:
![git1](https://cloud.githubusercontent.com/assets/8463461/13546097/1fd7b0b0-e2c9-11e5-8ffc-4d927880bdd7.png)

Light:
![git2](https://cloud.githubusercontent.com/assets/8463461/13546101/38322d34-e2c9-11e5-8497-5eee8066cbac.png)

**Comment by asmeurer** [comment]
That's a simple solution, but I think it would look better if we could detect the qtconsole background color and set the foreground color dynamically (and keep the background color transparent).

**Comment by articuno12** [comment]
Is this issue still open? Can I work on it?

**Comment by Flying-Pegasus** [comment]
@asmeurer is this issue still open, I want to work on it
